keyboards/v60_type_r/keymaps/followingghosts/keymap.c

```diff
@@ -0,0 +1,135 @@
+         * |-----------------------------------------------------------|
+         * |        |BLD|BLT|BLI|   |VolD|VolU|Mut|End|PgD|Dwn|  RESET |
+         * |-----------------------------------------------------------|
+         * |    |    |    |                        |    |AGSW|    |    |
+         * `-----------------------------------------------------------'
+         */
+
+  [1] = LAYOUT_60_ansi(
+	       KC_GRV,    KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,     KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,   KC_F12,  KC_DEL, \
+	       KC_TRNS,   KC_TRNS,  KC_UP,    RGB_TOG,  RGB_RI,  RGB_GI,  RGB_BI,   KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_UP,    KC_TRNS,          KC_INS, \
+	       KC_TRNS,   KC_LEFT,  KC_DOWN,  KC_RIGHT, RGB_RD,  RGB_GD,  RGB_BD,   KC_TRNS, KC_HOME, KC_PGUP, KC_LEFT, KC_RIGHT,          KC_TRNS,          \
+	       KC_TRNS,   BL_INC,  BL_STEP,  BL_DEC,  KC_TRNS,  KC_VOLD,   KC_VOLU, KC_MUTE, KC_END,  KC_PGDN,  KC_DOWN,           RESET,  \
+	       KC_TRNS,   KC_TRNS,  KC_TRNS,            KC_TRNS,                                                           KC_TRNS,  AG_SWAP, KC_TRNS, KC_TRNS),
+
+/*
+Mouse Layer
+M1 = Mouse Button 1
+M2 = Mouse Button 2
+WASD are Up Left Right Down respectively
+* ,-----------------------------------------------------------.
+* |   |   |    |   |   |   |   |   |   |   |   |    |    |    |
+* |-----------------------------------------------------------|
+* |     | M1|MUP|M2 |   |   |   |   |   |   |   |    |   |    |
+* |-----------------------------------------------------------|
+* |       |ML |MDN|MR |   |   |   |   |   |   |    |   |      |
+* |-----------------------------------------------------------|
+* |        |   |   |   |   |   |   |   |   |   |   |          |
+* |-----------------------------------------------------------|
+* |    |    |    |                         |    |    |   |    |
+* `-----------------------------------------------------------'
+*/
+         [2] = LAYOUT_60_ansi(
+                KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,     KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,  KC_TRNS,   KC_TRNS,  KC_TRNS, \
+                KC_TRNS,   KC_BTN1,  KC_MS_U,    KC_BTN2,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS,          KC_TRNS, \
+                KC_TRNS,   KC_MS_L,  KC_MS_D,  KC_MS_R, KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,          \
+                KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS,  KC_TRNS,  KC_TRNS,           RESET,  \
+                KC_TRNS,   KC_TRNS,  KC_TRNS,            KC_TRNS,                                                           KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS),
+
+
+};
+
+
+
+
+
+
+
+const uint16_t PROGMEM fn_actions[] = {
+
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
```

Comment:
Could you remove the `action_get_macro` function?  It's been deprecated in favor of `process_record_user`.